localization/english/wc_nicknames_l_english.yml

```diff
@@ -70,7 +70,19 @@
  nick_the_thunderous:0 "the Thunderous"
  nick_the_lightning_fist:0 "the Lightning Fist"
  nick_the_thunder_king:0 "the Thunder King"
- nick_the_thunder_desc:1 "[CHARACTER.GetShortUINameNoTooltipNoFormat|U] managed to slay a Titan Keeper. From this act, [CharYouSheHe(CHARACTER)] gained not only his power but that of Aman'thul himself over wind and storm."
+ nick_the_thunder_king_desc:0 "Lightning strikes in an instant and is over in a flash. But thunder. Thunder! Thunder proclaims the coming of the storm. Thunder quakes the skies long before the lightning strikes, and thunder echoes in the hills long after lightning's power is spent. It is thunder that sends animals cowering and fills the hearts of peasants with dread. Let thunder be my herald, so that my power is felt throughout the land. I will be... [the_thunder_king|E]!"
+ nick_the_lorewalker:0 "the Lorewalker"
```

Comment:
It could simply be named just "Lorewalker" as this is how he's referred as in WoW.
To make nicknames appear before the name you can put `is_prefix = yes` in the code.

Reply:
I didnt realize we had prefixes! sweet